### PR TITLE
feat: added support to configure the default explorer viz

### PR DESCRIPTION
**Pull request by cccs-jc**
Added support for configuring what the default visualization used by the chart explorer.

### CODE REVIEW

**Comment by junlincc** [comment]
@cccs-jc thanks for the contribution! could you provide more details what this change is for, in PR description? thanks!

**Comment by cccs-jc** [comment]
We are building a custom visualization based on agGrid and would like this custom table to be the default visualization when a user starts a chart explorer.

This also enables us to control the default row limit and whether to start in aggregation or raw query mode.

**Comment by cccs-jc** [comment]
@junlincc did that answer your question?

**Comment by junlincc** [comment]
@cccs-jc thanks for the answer, is this change going to affect current plug-in structure? are you building the custom viz using dynamic viz plug-in? 

cc @suddjian

**Review by suddjian — APPROVED** [review]
This looks good. Shouldn't affect any existing features @junlincc. Just a config option for what viz is the default in the viz picker.

File: superset/config.py [hunk 1]
```diff
@@ -114,6 +114,9 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 VERSION_SHA_LENGTH = 8
 VERSION_SHA = _try_json_readsha(VERSION_INFO_FILE, VERSION_SHA_LENGTH)
 
+# default viz used in chart explorer
+DEFAULT_EXPLORER_VIZ = 'table'
```
Comment (villebro) [inline superset/config.py:118]: nit: I would prefer to call this `DEFAULT_VIZ_TYPE`

**Review by villebro — COMMENTED** [review]
This is a good idea - a minor improvement proposal, other than that LGTM

File: superset/config.py (hunk 1 above)
Comment (cccs-jc) [reply superset/config.py:118]: okay sounds good I can change that.

**Comment by cccs-jc** [comment]
@villebro Did the change you suggested. Renamed config to DEFAULT_VIZ_TYPE

**Comment by villebro** [comment]
@cccs-jc LGTM, but you need to fix the linting error: 
```
/home/runner/work/superset/superset/superset-frontend/src/explore/store.js
  44:18  error  Insert `⏎···`  prettier/prettier
```

**Comment by cccs-jc** [comment]
@villebro I'm not sure what the linter is complaining about. I check the line break in case it might be a windows line break. But looks good to me..

**Comment by suddjian** [comment]
The issue is Prettier's formatting guidelines says the line is too long and wants it broken up. You can run `npm run format` to auto-fix.

**Comment by ktmud** [comment]
It would be great to allow setting this per user profile! (another reason to add meta storage for user configs)

**Pull request merged**